scripting/testsuite/tf2itemsinfo_testkv.sp

```diff
@@ -1,15 +1,18 @@
-#pragma semicolon 1
-
-#include <sourcemod>
-#include <tf2>
-#include <tf2itemsinfo>
-
-public OnPluginStart() {
-	if (!TF2II_IsItemSchemaPrecached()) return;
-	new Handle:kv = TF2II_GetAttribKeyValues( 694 );
-	decl String:strFilePath[PLATFORM_MAX_PATH] = "data/tf2itemsinfo22.txt";
-	BuildPath( Path_SM, strFilePath, sizeof(strFilePath), strFilePath );
-	LogError("Written to %s", strFilePath);
-	KeyValuesToFile(kv, strFilePath);
-	CloseHandle(kv);
+#pragma semicolon 1
+#pragma newdecls required
+
+#include <sourcemod>
+#include <tf2>
+#pragma newdecls optional
+#include <tf2itemsinfo>
+#pragma newdecls required
+
+public void OnPluginStart() {
+	if (!TF2II_IsItemSchemaPrecached()) return;
+	Handle kv = TF2II_GetAttribKeyValues( 694 );
```

Comment:
you can't downcast ``Handle`` to ``KeyValues``, but I could add a view_as if necesary